es/quickstart.md

```diff
@@ -0,0 +1,243 @@
+
+Para ejecutar el ejemplo, pulse con el bot&oacute;n derecho sobre &eacute;l y seleccione el bot&oacute;n verde en el men&uacute; desplegable. Ver&aacute;, en la ventana de abajo de IntelliJ, una serie de resultados. El n&uacute;mero m&aacute;s a la derecha es el resultado de error de las clasificiaciones de la red. Si su red est&aacute; aprendiendo, entonces ese n&uacute;mero decrecer&aacute; a lo largo del tiempo con cada bloque que procesa. Al final, la ventana le mostrar&aacute; cu&aacute;nto de precisa su modelo de red neuronal se ha vuelto:
+
+![mlp classifier results](../img/mlp_classifier_results.png)
+
+En otra ventana, aparecer&aacute; un gr&aacute;fico, mostr&aacute;ndo como la red perceptr&oacute;n multicapa (MLP en ingl&eacute;s) ha clasificado los datos del ejemplo. Deber&iacute;a parecerse a esto:
+
+![mlp classifier viz](../img/mlp_classifier_viz.png)
+
+¡Enhorabuena! Acaba de entrenar su primera red neuronal con Deeplearning4j. Ahora, por qu&eacute; no prueba nuestro siguiente tutorial: [**MNIST para Principiantes**](./mnist-for-beginners), donde aprender&aacute; c&oacute;mo clasificar im&aacute;genes.
+
+## Siguientes pasos
+
+1. &Uacute;nete a nosotros en Gitter. Tenemos tres canales con una gran comunidad.
+  * [DL4J Live Chat](https://gitter.im/deeplearning4j/deeplearning4j) es el canal principal para todo lo relacionado con DL4J. La mayor parte de la gente se junta ah&iacute;.
+  * [Tuning Help](https://gitter.im/deeplearning4j/deeplearning4j/tuninghelp)  es para gente que acaba de empezar con redes neuronales. Los principiantes por favor visitadnos aqu&iacute;!
+  * [Early Adopters](https://gitter.im/deeplearning4j/deeplearning4j/earlyadopters) es para aquellos que nos est&aacute;n ayudando en la revisi&oacute;n y mejorando la siguiente entrega. ATENCI&Oacute;N: Este es para la gente m&aacute;s experimentada.
+2. Lea la [introducción a las redes neuronales profundas](./neuralnet-overview) o [uno de nuestros detallados tutoriales](./tutorials).
+3. Eche un vistazo a la m&aacute;s detallada [Gu&iacute;a Extensa de Configuraci&oacute;n](./gettingstarted).Check out the more detailed [Comprehensive Setup Guide](./gettingstarted).
+4. Explore la [documentaci&oacute;n de DL4J](./documentation).
+5. **Gente de Python**: si tiene planeado evaluar el rendimiento de Deeplearning4j para compararlo con otro bien conocido framework de Python [x], por favor lea [estas instrucciones](https://deeplearning4j.org/benchmark) sobre c&oacute;mo optimizar el tama&ntilde;o del heap, el recolector de basura y ETL en la JVM. Sigui&eacute;ndolos, observar&aacute; una mejora de al menos *10x de velocidad en el tiempo de entrenamiento*.
```

Comment:
Esta caja (`[x]`) es problematica.

Reply:
Hi @huitseeker, the `[x]` box is also present in the original (english) quickstart.md file, should I remove it?.